### PR TITLE
docs: Parallel execution of Test

### PR DESCRIPTION
**Pull request by shubham-cmyk**
Signed-off-by: Shubham Gupta <iamshubhamgupta2001@gmail.com>

Fix : https://github.com/kyverno/chainsaw/issues/619

### CODE REVIEW

**Comment by eddycharly** [comment]
Thanks @shubham-cmyk !

This can be controlled at the test level too, with the `concurrent` field.
All non concurrent tests will be run first, then concurrent tests next.

**Comment by shubham-cmyk** [comment]
Just for the reference of the PR

The total test at CPU can be controlled by setting `GOMAXPROCS`
 https://pkg.go.dev/runtime#GOMAXPROCS

**Comment by eddycharly** [comment]
Last request, can you add an entry in the release notes and set labels accordingly ?

**Comment by shubham-cmyk** [comment]
@eddycharly do you mean labels on the PR ?

File: .release-notes/v0.0.1.md [hunk 1]
```diff
@@ -7,3 +7,7 @@ Release notes for `v0.0.1`.
 - Initial MVP release with support for `Apply`, `Delete`, `Assert` and `Error` operations
 - Supports configuration from both config file and command line flags
 - Full documentation available at https://kyverno.github.io/chainsaw
+
+## 📚 Docs 📚
+
+- Parallel execution of Test
```
Comment (eddycharly) [inline .release-notes/v0.0.1.md:13]: This should be in `main.md` (not part of v0.0.1)

Comment (shubham-cmyk) [reply .release-notes/v0.0.1.md:13]: okay let me fix it

**Review by eddycharly — APPROVED** [review]

**Pull request merged**